libraries/Microsoft.Bot.Builder.Dialogs.Debugging/Protocol/Next.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Dialogs.Debugging.Protocol
+{
+#pragma warning disable CA1716 // Identifiers should not match keywords (by design and we can't change it without breaking binary compat)
```

Comment:
not shipped component, break it.

Reply:
Leaving it for now, we can change it when we make it public.